Android.bp

```diff
@@ -0,0 +1,45 @@
+// Copyright 2020 The Android Open Source Project
+
+cc_defaults {
+    name: "exfat-utils-defaults",
+
+    cflags: [
+        "-DEXFAT_TOOLS_VERSION=\"1.0.1\"",
+    ],
+    local_include_dirs: [
```

Comment:
Could you separate this Android.bp and create Android.bp  in each sub-directory?

Reply:
This is how most of the external projects in android are handled, top makefile with every possible usable entry ( f2fs-tools, mkfs dos and other non da related utils )
Also it would make the defaults a bit harder to reach, that's why everything is in the same level given the shared dependencies